code/game/machinery/computer/dna_console.dm

```diff
@@ -644,6 +657,64 @@
 
 			return
 
+		//Activate mutation already in a subject's genome
+		if("activate_mutation")
+			if(!can_modify_occupant())
+				return
+
+			var/path = GET_MUTATION_TYPE_FROM_ALIAS(params["alias"])
+			if(stored_research && (path in stored_research.discovered_mutations))
```

Comment:
Not sure if looking for it in list everytime is a good idea.
Make discovered mutations be associative list and just check index